src/reducer.js

```diff
@@ -774,6 +763,25 @@ export const onPushApiFailed = (state) => ({
   },
 });
 
+/**
+ * @param {Boolean} action.payload.error The captured Error object
+ * @param {Boolean} action.payload.isFatal Indicates if the error is fatal
+ */
+export const onExceptionCaptured = (state, { payload }) => {
```

Comment:
I guess this does not belong to this PR, it is already in `dev`, can you rebase?